state/subnets.go

```diff
@@ -223,12 +226,11 @@ func (s *Subnet) setSpace(subnets mongo.Collection) error {
 	return nil
 }
 
-// Update adds new info to the subnet based on provided info.  Currently no
-// data is changed, unless it is the default space from MAAS.  There are
-// restrictions to the additions allowed:
-//   - no change to CIDR, more work to determine how to handle needs to
-// be done.
-//   - no change to ProviderId nor ProviderNetworkID, these are immutable.
+// Update adds new info to the subnet based on input info.
+// Currently no data is changed unless it is the default space from MAAS.
```

Comment:
s/default space from MAAS/undefined space from MAAS/

Based on conversations yesterday and confusion over whether MAAS has a default space or not.